src/ApolloProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import { ApolloClient } from 'apollo-client'
+import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory'
+import { HttpLink } from 'apollo-link-http'
+import { onError } from 'apollo-link-error'
+import { ApolloLink } from 'apollo-link'
+import { ApolloProvider as ReactApolloProvider } from 'react-apollo'
+import introspectionQueryResultData from 'utils/fragmentTypes.json'
+
+const fragmentMatcher = new IntrospectionFragmentMatcher({
+  introspectionQueryResultData,
+})
+
+const client = new ApolloClient({
+  link: ApolloLink.from([
+    onError(({ graphQLErrors, networkError }) => {
+      if (graphQLErrors)
+        graphQLErrors.map(({ message, locations, path }) =>
+          console.log(`[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`)
+        )
+      if (networkError) console.log(`[Network error]: ${networkError}`)
+    }),
+    new HttpLink({
+      uri: '/crispy-api',
```

Comment:
have you tried how it will work with API?
I think, `api` name would be better

Reply:
I will change all URL's once the crispy-api endpoint would be ready

Reply:
I guess we should move it to config, and get `uri` from config